test/integration/api/workflow.spec.js

```diff
@@ -1949,6 +2070,93 @@ test('removes a workflow', async (t) => {
     .expect(404)
 })
 
+test('list workflow logs', async (t) => {
+  const authorizationHeaders = await getAccessTokenHeaders({ t, permissions: ['workflowLog:list:all'] })
+
+  const { body: obj } = await request(t.context.serverUrl)
+    .get('/workflow-logs?page=2')
+    .set(authorizationHeaders)
+    .expect(200)
+
+  t.true(typeof obj === 'object')
+  t.true(typeof obj.nbResults === 'number')
+  t.true(typeof obj.nbPages === 'number')
+  t.true(typeof obj.page === 'number')
+  t.true(typeof obj.nbResultsPerPage === 'number')
+  t.true(Array.isArray(obj.results))
+})
+
+test('list workflow logs with id filter', async (t) => {
+  const authorizationHeaders = await getAccessTokenHeaders({ t, permissions: ['workflowLog:list:all'] })
+
+  const { body: { results: workflowLogs } } = await request(t.context.serverUrl)
+    .get('/workflow-logs')
+    .set(authorizationHeaders)
+    .expect(200)
+
+  const workflowLog = workflowLogs[0]
+
+  const { body: obj } = await request(t.context.serverUrl)
+    .get(`/workflow-logs?id=${workflowLog.id}`)
+    .set(authorizationHeaders)
+    .expect(200)
+
+  t.is(typeof obj, 'object')
+  t.is(obj.nbResults, 1)
+  t.is(obj.nbPages, 1)
+  t.is(obj.page, 1)
+  t.is(typeof obj.nbResultsPerPage, 'number')
+  t.is(obj.results.length, 1)
+})
+
+test('list workflow logs with advanced filters', async (t) => {
```

Comment:
`type` filter testing is also needed.

Ideally you could write erroneous code in a workflow to generate a `runError` log.